css/amp.css

```diff
@@ -484,14 +484,18 @@ i-amphtml-scroll-container.amp-active, i-amp-scroll-container.amp-active {
 /**
  * `i-amphtml-loading-container`, `i-amphtml-loader` and `i-amphtml-loader-dot` all support
  * a "loading indicator" usecase. `i-amphtml-loading-container` is mostly responsible
- * for alighning the loading indicator, while `i-amphtml-loader` and
+ * for aligning the loading indicator, while `i-amphtml-loader` and
  * `i-amphtml-loader-dot` are an implementation for a default loading indicator. The
  * default implementation includes the three-dot layout and animation.
  */
 .i-amphtml-loading-container.amp-hidden {
   visibility: hidden;
 }
 
+amp-list[auto-resize] > .i-amphtml-loading-container.amp-hidden {
```

Comment:
can you put a comment here that explains `amp-hidden` changes visibility which still takes space but in the case where `amp-list` becomes a container we want to fully hide the loader and not let it take space.

Reply:
Done.